src/DocumentTemplate/cDocumentTemplate.c

```diff
@@ -960,7 +960,7 @@ static struct PyMethodDef Module_Level__methods[] = {
    "join a list of plain strings into a single plain string,\n"
    "a list of unicode strings into a single unicode strings,\n"
    "or a list containing a mix into a single unicode string with\n"
-   "the plain strings converted from latin-1"},
+   "the plain strings converted from UTF-8"},
```

Comment:
If we do this, we need to update the [Python reference implementation](https://github.com/zopefoundation/DocumentTemplate/blob/cb27035473a610dc78c630669a29241935947d7a/src/DocumentTemplate/_DocumentTemplate.py#L140) as well.

Reply:
Updated just now in 1eedab1dbfa1c57f4e40b5397f207759ef8e0d01.